app/assets/javascripts/components/overview/my_exercises/components/Header.jsx

```diff
@@ -5,29 +5,30 @@ import { HashLink as Link } from 'react-router-hash-link';
 export const Header = ({ completed = false, course, remaining = [], text }) => (
   <header className="header">
     <h3 className={completed ? 'completed' : ''}>
-      { text }
-      {
-        remaining.length
-        ? <small>{remaining.length} additional exercises remaining.</small>
-        : null
-      }
+      {text}
+      {remaining.length ? (
+        <small>
+          {remaining.length} {I18n.t('training.remaining_exercise')}
```

Comment:
Rather than concatenating the number with the message, this should be implemented as an argument to the message, so that translators have full control of how to handle the number within the message.